app/views/firms/partials/_firm_details.html.erb

```diff
@@ -16,6 +16,16 @@
 
 <%= heading_tag(t('firms.show.panels.firm.services.heading'), level: 3, class: 'l-firm__heading') %>
 
+<%= heading_tag(t('firms.show.panels.firm.services.types_of_advice.heading'), level: 4) %>
+
+<ul class="types-of-advice__list">
```

Comment:
Good solution. How about creating an array of advice types which you can iterate over?